.storybook/3.COLORS.stories.mdx

```diff
@@ -111,7 +111,6 @@ var(--color-success-default)
 var(--color-success-alternative) [DEPRECATED]
 var(--color-success-muted)
 var(--color-success-inverse)
-var(--color-success-disabled) [DEPRECATED]
```

Comment:
No `success-disabled` left in the codebase

![Screenshot 2024-06-06 at 1 59 28 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/0394ac5a-46fe-4b84-90ba-b0215ce2fa33)